schemas/campaigns/activity.schema.json

```diff
@@ -34,6 +34,65 @@
           "type": "string",
           "description": "Activity title."
         },
+        "xdm:brief": {
+          "title": "Brief",
+          "type": "string",
+          "description": "An HTML brief describing the activity."
+        },
+        "xdm:thumbnail": {
+          "title": "Thumbnail",
+          "type": "string",
+          "description": "Thumbnail image representing the activity."
+        },
+        "xdm:channel": {
+          "title": "Channel",
+          "enum": [
+            "not_specified",
+            "email",
+            "sms",
+            "push_notification",
+            "direct_mail"
+          ],
+          "meta:enum": {
+            "email": "email",
+            "sms": "SMS",
+            "push_notification": "Push Notification",
+            "not_specified": "Not Specified"
+          },
+          "description": "Channel on which this activity is executed.",
+          "default": "not_specified"
+        },
```

Comment:
Lets try to unify on channel type definitions, statuses, and destination information that will work well for Engage, Campaign and Offer Management. In Engage, an activity has a type (which is really the channel type, "twitter", "facebook", or channels we currently can't post to like "article" or "blog post"). This is primarily used to control what type of template can be used, and what type of destinations it can be published to. An activity then has multiple destinations it can be published to. In the case of social publishing, which is all that we now support, those are social network accounts. 

For status, we track two dimensions of status: activity status and publishing status. Activity status is about tracking the activity through draft, editing, review and approval. Publishing status is an array and tracks the delivery status for each publishing destination.

Seems like we both agree that channel and activity status belong here. We'll need to agree on what the enumeration values should be, but I don't honestly care that much. The real question is what to do about the publishing destinations and statuses. We have two options: leave that information in the Engage extensions, or move it into the core. If we move it into the core, do we think about how those fields would apply to Campaign & Offer Management. I'm not totally sure what the right answer is.

Reply:
Yes, this makes sense. Actually, we have similar concepts:
- We do not have approvals in ACS yet, but need it. That's a good fit for the activity status
- We do have a publication notion (that we called analysis for deliveries). A "published" delivery is read-only in term of edition. I'm good to separate activity and publishing status, and actually prefer that.
- We do have a notion of destination, that we call routing. The only difference is that we can only route to a single destination. Routing in ACS could be Campaign MTA (meaning that we send the messages) or flat files (meaning that we generate files for a third party to deliver). The latter is usually used for direct mail which we do not deliver ourselves.

So it looks like it's core to me:
- activity status would be draft, editing, review and approval. Campaign would only use editing.
- publishing status would be ready_to_start, started, finished, failed
- destination as you described. For Campaign, this would be primarily driven by the channel (direct mail => file ; email, push => MTA)